### DISCUSSION TITLE
Bitwarden server with HTTPS

### DISCUSSION THREAD

**Discussion by t0msk** [dani-garcia/vaultwarden#2734] [Q&A]
Hello, I have RPi 4 at home and I would like to run Bitwarden server on it, I use Portainer for managing docker containers (every service I run on that RPi 4 is container).

So I followed this tutorial: [Self Hosting Bitwarden on the Raspberry Pi - Pi My Life Up 1](https://pimylifeup.com/raspberry-pi-bitwarden/)

And it says that I have to install nginx on host RPi 4 for HTTPS, because Bitwarden image on docker hub is HTTP only and some functions wouldn’t work with HTTP (and it is insecure), **so I would like to know if there is some Bitwarden image with HTTPS support inside it, or if I can somehow install Nginx in that container and make it persistent.**

Because I don’t like idea installing any service (Nginx) outside containers.

Thank you

**Comment by lukasdotcom** [comment]
You can use [NGINX Proxy Manager](https://nginxproxymanager.com/). It is a docker container.
